### DISCUSSION TITLE
With Postgres connection mode set to "Simple", can't add or delete columns from table

### DISCUSSION THREAD

**Discussion by DeflateAwning** [dbeaver/dbeaver#21438] [General]
### Description

When you create a Postgresql connection, and set its `Connection view` to "Simple", DBeaver doesn't let you add or remove columns in the table view. 

### DBeaver Version

Community Edition 23.2.0

### Operating System

Ubuntu

### Database and driver

```
PostgreSQL 12.14
PostgreSQL JDBC Driver 42.5.2
```

**Comment by LonwoLonwo** [comment]
Hello @DeflateAwning 

Yes, that's expected behavior for the "Simple View". Other object creation also is not available via UI. 
You can choose a custom or advanced navigator view or change your tables via SQL Editor.

**Comment by DeflateAwning** [comment]
Is there a way to show only tables (or only tables and views) in the sidebar without decreasing the capabilities of DBeaver overall?

**Comment by LonwoLonwo** [comment]
Unfortunately, there is no way to do it like this. 
Please use custom mode or advanced to change metadata via UI.

**Reply by DeflateAwning** [reply]
How can I do that?

**Reply by AlexZeitler** [reply]
@DeflateAwning did you solve this?

**Reply by DeflateAwning** [reply]
No I did not

**Reply by LonwoLonwo** [reply]
![2023-11-09 10_41_11-DBeaver Ultimate 23 2 4 - _MariaDB 10 6_ Script-99](https://github.com/dbeaver/dbeaver/assets/45152336/2edebea8-6af1-4b8b-bfed-669b5d0c89c6)

You can do it like this. Advanced or Custom navigator views will help you with UI metadata modification.

**Reply by AlexZeitler** [reply]
@LonwoLonwo Is it possible to keep the simple structure but have the "delete table" command available using custom navigator views? Tried Custom views already but didn't seem to work.

**Reply by LonwoLonwo** [reply]
Could you please provide your DBeaver version, database name, and a screen?

**Reply by AlexZeitler** [reply]
Version `23.2.4.202311051706`

![image](https://github.com/dbeaver/dbeaver/assets/287480/d2f30e58-94b8-422c-966d-ea3f73c70c26)

**Reply by LonwoLonwo** [reply]
Simple view is only for viewing. Use other Connection View types for editing.

**Reply by AlexZeitler** [reply]
Thanks for the clarification

**Reply by mlarcher** [reply]
When in advanced mode, the left panel is cluttered with a verbose tree structure that slows down navigation quite a bit for a value I fail to idnetify. 

I'd really love to have the same structure as in simple view, but still be able to delete a table or alter columns structure.

If changing those is not possible in simple mode, could we see it the other way around and provide a way to get a simple tree structure when using advanced mode ?

For the record, I also couldn't figure out how to have a table delete entry in the contextual menu when in custom mode...
